libraries/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ set(BN256_INSTALL_COMPONENT "dev")
 
 set( Boost_USE_MULTITHREADED      ON )
 set( Boost_USE_STATIC_LIBS ON CACHE STRING "ON or OFF" )
+# don't include boost mysql library as it does a find_package(OpenSSL) thus finding the system openssl which could conflict with the bundled boringssl
+set( BOOST_EXCLUDE_LIBRARIES "mysql" )
```

Comment:
This is something we'll need to be mindful of on boost bumps (or really any bundled dependency).